addons/binding/org.openhab.binding.camera/.classpath

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry exported="true" kind="lib" path="lib/commons-codec-1.10.jar"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7"/>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry kind="src" path="src/main/java"/>
+	<classpathentry exported="true" kind="lib" path="lib/httpcore-4.4.6.jar"/>
```

Comment:
Would be great to depend on the already provided Jetty instead...

We also have a wrapper that support fetching of images out of the box: 
https://www.eclipse.org/smarthome/documentation/javadoc/org/eclipse/smarthome/io/net/http/HttpUtil.html

Reply:
The reason for using the original apache lib here is the special authentication support which I can not see in your HttpUtil. The client really needs all the libs referenced in the manifest.

Reply:
As far I quickly saw this code is just a complex way to add a header:

```
Authorization: Basic base64encode(user+":"+password)
```

So if you would be helped if we add support for additional headers to the `HttpUtil` I would be happy not dragging along these extra dependencies. @kaikreuzer given our pending `HttpUtil` discussion wdyt?

Source: https://en.wikipedia.org/wiki/Basic_access_authentication